src/main/java/com/dev/cinema/controllers/AuthenticationController.java

```diff
@@ -0,0 +1,23 @@
+package com.dev.cinema.controllers;
+
+import com.dev.cinema.model.dto.UserRequestDto;
+import com.dev.cinema.security.AuthenticationService;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
+
+@RestController
+@RequestMapping
```

Comment:
do we need this annotation here?

Reply:
Test without it and see that it is unnecessary, thanks.